server.py

```diff
@@ -12,16 +12,26 @@
 
 db = SqliteDatabase(DATABASE_NAME)
 
-
 class User(Model):
+    """ 
+    Defines username and password for individual user.
+    """
     username = CharField(unique=True)
     password = CharField()
 
     class Meta:
+        """
+        define the database in the main class
+        """
         database = db
 
 
 class Session(Model):
+    """
```

Comment:
see, this comment does not tell me anything new. What it says I can read in the code. A good docstring would let me know WHY the string being random is necessary.